dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.netty-1.0.feature

```diff
@@ -1,18 +1,15 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=io.openliberty.netty-1.0
```

Comment:
Since we're moving this to a private feature we need to update the feature name to follow the open liberty naming guidelines: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-naming-guidelines

Reply:
I'll update to be io.openliberty.netty.internal-1.0